### PR TITLE
Grant report

### PR DESCRIPTION
**Pull request by slonigiraf**
https://github.com/w3f/Grants-Program/blob/master/applications/slonigiraf.md

# Milestone Delivery Checklist

- [x] The [milestone-delivery-template.md](https://github.com/w3f/Grant-Milestone-Delivery/blob/master/deliveries/milestone-delivery-template.md) has been copied and updated.
- [x] The [invoice form :pencil:](https://forms.gle/LSRr7PCjBpEbKGh89) has been filled out for this milestone.
- [x] This pull request is being made by the same account as the accepted application.
- [x] In case of acceptance, the payment will be transferred to the BTC/ETH payment address provided in the application.
- [x] The delivery is according to the [Guidelines for Milestone Deliverables](https://github.com/w3f/Grants-Program/blob/master/docs/milestone-deliverables-guidelines.md).

Link to the application pull request: https://github.com/w3f/Grants-Program/pull/733 < please fill this in with the PR number of your application.

### CODE REVIEW

**Comment by semuelle** [comment]
Thank you for the delivery, @slonigiraf. We will look into it as soon as possible. We have a bit of a backlog currently, so please allow a week or two for us to get back to you.

File: deliveries/slonigiraf-milestone-1.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Milestone Delivery :mailbox:
+
+**The [invoice form :pencil:](https://docs.google.com/forms/d/e/1FAIpQLSfmNYaoCgrxyhzgoKQ0ynQvnNRoTmgApz9NrMp-hd8mhIiO0A/viewform) has been filled out correctly for this milestone and the delivery is according to the official [milestone delivery guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/milestone-deliverables-guidelines.md).**  
+
+* **Application Document:** https://github.com/w3f/Grants-Program/blob/master/applications/slonigiraf.md
+* **Milestone Number:** e.g. 1
```
Comment (semuelle) [inline deliveries/slonigiraf-milestone-1.md:6]:
```suggestion
* **Milestone Number:** 1
```

Comment (slonigiraf) [reply deliveries/slonigiraf-milestone-1.md]: Fixed

**Comment by hakan-w3f** [comment]
Hi, I'd like to inform you that the evaluation has been successfully completed. As mentioned earlier on our email correspondence, there's some room for improvement on the UI regarding how it responds to profile changes (i.e. switching from Alice to Bob etc.), but other than that we're happy with all the deliverables. Looking forward to seeing what you build next with Substrate. Kind regards.

**Review by Noc2 — APPROVED** [review]

**Comment by meldien** [comment]
Hi @slonigiraf 
I noted the payment details are missing. If we have to pay in FIAT please attache on the invoice all bank details. If we have to pay in Crypto please insert your crypto address. 
Many thanks for your update. 
Best wishes,
Melanie

**Comment by slonigiraf** [comment]
@fededubbi @meldien I've submitted the updated invoice today. Thank you very much.

**Comment by RouvenP** [comment]
hi @slonigiraf, we transferred the payment today. Thanks!

**Pull request merged**